client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -82,22 +84,11 @@ function usePerMonthDescription( {
 				? formatCurrency( planPrices.rawPrice, currencyCode, { stripZeros: true } )
 				: null;
 
-		// TODO: Remove check once text is translated
-		const displayNewPriceText =
-			isEnglishLocale ||
-			( i18n.hasTranslation( 'per month, %(rawPrice)s billed annually, Excl. Taxes' ) &&
```

Comment:
**Note:**

We no longer need to check whether or not translations have been localized. They were completed on May 16th https://github.com/Automattic/wp-calypso/pull/76738#issuecomment-1550128637